### PR TITLE
A default card to replace a carousel of joined clubs

### PR DESCRIPTION
**Pull request by haleyjung**
- Stats render issue fixed once again
- A default card is added to replace an empty carousel section when a user has not joined any clubs. 
Screen Shot 2022-05-13 at 6 23 45 PM

### CODE REVIEW

**Comment by addisonhernandez** [comment]
:star_struck: This might be the most beautiful PR comment I've seen yet.

File: client/pages/BookClubs/index.jsx [hunk 1]
```diff
@@ -130,28 +130,50 @@ export default function BookClubs() {
...
-            <button onClick={handlePrev} type="button">
-              <ArrowBackIosIcon />
-            </button>
-          </div>
-          <div className="next">
-            <button onClick={handleNext} type="button">
-              <ArrowForwardIosIcon />
-            </button>
-          </div>
+          {myBookClubs.length === 0 ? (
```
Comment (addisonhernandez) [inline client/pages/BookClubs/index.jsx:133]: Is it ever possible for `myBookClubs` to be `null` or `undefined`?

Comment (addisonhernandez) [reply client/pages/BookClubs/index.jsx:133]: If so, I recommend replacing `myBookClubs.length` with `myBookClubs?.length`

**Comment by haleyjung** [comment]
**Proposal:** removing reading stats from user's dropdown menu to logged in user's navigation bar. This won't trigger the render bug with the menu item.

Screen Shot 2022-05-13 at 6 42 03 PM

File: components/NavBar/AccountMenu.jsx [hunk 2]
```diff
@@ -94,13 +94,6 @@ export default function AccountMenu() {
         <MenuItem>
           <Avatar /> My account
         </MenuItem>
-        <MenuItem>
-          <Avatar />
-          <Link to="/stats" style={{ 'text-decoration': 'none', color: 'black' }}>
-            {' '}
-            My Reading Stats{' '}
```
Comment (addisonhernandez) [inline components/NavBar/AccountMenu.jsx:101]: :crying_cat_face: We're removing the stats from the NavBar?

Comment (haleyjung) [reply components/NavBar/AccountMenu.jsx:101]: Proposing to remove it from the user icon dropdown menu and move to the navigation bar

File: client/pages/BookClubs/index.jsx (hunk 1 above)
Comment (haleyjung) [reply client/pages/BookClubs/index.jsx:133]: myBookClubs state gets set with `data.results` returned from myBookclubs endpoint which always returns an empty array when there's no value.

**Pull request merged**